reflex/components/radix/themes/color_mode.py

```diff
@@ -182,9 +166,7 @@ class ColorModeNamespace(BaseVar):
     """Namespace for color mode components."""
 
     icon = staticmethod(ColorModeIcon.create)
-    switch = staticmethod(ColorModeSwitch.create)
```

Comment:
if we're removing switch from the exposed namespace, we should probably remove the `ColorModeSwitch` class as well